### PR TITLE
275 Change Docs to Documentation in the top navigation

### PR DESCRIPTION
**Pull request by qjac**
Closes https://github.com/GSA/openacr/issues/275

QA
- Documentation is spelled correctly
- The 'Documentation' nav item links to  https://gsa.github.io/openacr-editor/chapter/support_documentation_and_services

### CODE REVIEW

**Review by mgifford — APPROVED** [review]
Looks good to me. Not sure if any other testing is required beyond going here https://gsa.github.io/openacr-editor/chapter/support_documentation_and_services
Screen Shot with documentation change in header

**Comment by dmundra** [comment]
@qjac, agreed with Mike changes are looking good. If you can please add the issue URL to the description and if you have any QA steps outside what Mike suggested.

Feel free to merge this after approval.

**Comment by mgifford** [comment]
There are lots of examples @qjac in the PR history to pull from that Daniel has added https://github.com/GSA/openacr-editor/pull/11 is an example. 

I don't think this one would have any "surprises" but by outlining what should be tested it makes it much easier for me to ensure I've tested that I see the work that was done and get the same results.

**Pull request merged**
